src/frontend/src/flows/addDevice/manage/verifyTentativeDevice.ts

```diff
@@ -48,116 +115,126 @@ const pageContent = (alias: string) => {
   });
 };
 
+export const verifyTentativeDevicePage = (
+  props: Parameters<typeof verifyTentativeDeviceTemplate>[0],
+  container?: HTMLElement
+): void => {
+  const contain =
```

Comment:
nth: we probably  have this `container ?? (document.getElementById("pageContent") as HTMLElement);` few times in code now, maybe some day worth a utils?

Reply:
Some day, like the prettier command...